### DISCUSSION TITLE
Other platforms' data import

### DISCUSSION THREAD

**Discussion by mickymix** [teslamate-org/teslamate#882] [Ideas]
**What would you like to be added**: I would appreciate the chance to import TezLab's exports, if possible.

**Why is this needed**: I would prefer moving my data (old data too) to TeslaMate, which I prefer...

**Additional context**: I've tried importing data exported from TezLab (which is CSV indeed), but I think there are differences between TeslaFi's format and TezLab's as it finds no data to import.

**Comment by LelandSindt** [comment]
If you know or don't mind learning a little python this: https://github.com/adriankumpf/teslamate/issues/452 thread might give you what you are looking for to convert the data to TeslaFi format for import into Teslamate.

**Comment by mickymix** [comment]
Thank you for the really precious and interesting link to your work: it is indeed very interesting and helpful.

What I would need anyway is much more simpler (while at the same time I don't know how to do it): TezLab's exports are definitely less descriptive of car's statuses, as they simply contain a CSV list of all the drives (from point A to point B, with a few other details, but not so much) and a second CSV list of all the charges (started at, ended at, and another few other details, not so much again).

Honestly I could also consider adding them manually, if this could be possible.

I've seen in another post that someone has been succesfull adding the manually... within SQL.
I would prefer adding them manually on the UI instead, to avoid damages to the DB (which I'm surely capable of...).
:-D

**Comment by shanhard** [comment]
Did you have any luck getting your Tezlab data into Teslamate?

**Reply by mickymix** [reply]
Sorry, no.
I actually tried to investigate this for a couple of days, then decided the effort was not worth it, for me, so I gave up.
